geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegion.java

```diff
@@ -1221,10 +1223,21 @@ public void addGatewaySenderId(String gatewaySenderId) {
     }
   }
 
+  /**
+   * @deprecated as of 10.0 use
+   *             {@link PartitionedRegion#updatePRConfigWithNewSetOfAsynchronousEventDispatchers(Set)}
+   *             instead
+   */
+  @Deprecated
   public void updatePRConfigWithNewSetOfGatewaySenders(Set<String> gatewaySendersToAdd) {
```

Comment:
We don't need to "deprecate" the methods as this class is in the internal package. What I was saying is that this method seems to be update the gateway sender id instead of AEQ ids, or am I wrong?

Reply:
The method, despite its confusing name, updates both the `gateway-sender` and `async-event-queue` (internally there's only **one single list** containing both types of dispatchers without any distinction), that's why I changed the name. I'll go ahead and remove the old versions instead of deprecating them.